src/context/Create.tsx

```diff
@@ -43,15 +46,39 @@ const setDestination = (
 
     const destination = getUrlParam("destination");
     if (urlParamIsSet(destination)) {
-        if (isValidForAsset(BTC, destination)) {
-            setAssetReceive(BTC);
+        const btcDestination = isValidForAsset(BTC, destination);
+        const lbtcDestination = isValidForAsset(LBTC, destination);
```

Comment:
Not a huge fan of having all chain currencies here explicitly. Would be nicer to iterate over everything we know to handle currencies we add it in the future without having to touch this code